ports/openssl/windows/portfile.cmake

```
@@ -29,6 +29,10 @@ if(DEFINED OPENSSL_USE_NOPINSHARED)
     set(CONFIGURE_OPTIONS ${CONFIGURE_OPTIONS} no-pinshared)
 endif()
 
+if(DEFINED OPENSSL_NO_AUTOLOAD_CONFIG)
```

```suggestion
if(OPENSSL_NO_AUTOLOAD_CONFIG)
```
In addition to testing if it's defined, we should also ensure it is a truthy value.

---

Done. Originally I copied it from the previous block which has `if(DEFINED OPENSSL_USE_NOPINSHARED)`.